pandas/io/tests/parser/test_parsers.py

```diff
@@ -84,13 +84,6 @@ def read_table(self, *args, **kwds):
 
 
 class TestPythonParser(BaseParser, PythonParserTests, tm.TestCase):
-    """
-    Class for Python parser testing. Unless specifically stated
-    as a PythonParser-specific issue, the goal is to eventually move
-    as many of these tests into ParserTests as soon as the C parser
-    can accept further specific arguments when parsing.
```

Comment:
This is explicitly written in the header of `pandas/io/tests/parser/python_parser_only.py`, so this comment here in `test_parsers` is just a duplicate.